### DISCUSSION TITLE
Tedana-related tasks for Brainhack Donostia 2023

### DISCUSSION THREAD

**Discussion by eurunuela** [ME-ICA/tedana#987] [Polls]
Hi there!

[Brainhack Donostia 2023](https://brainhack-donostia.github.io/) is around the corner (Nov 22-24) and that means I will take the opportunity to work exclusively on developing features for tedana.

I wanted to know what folks think would be nice to have incorporated into tedana before the end of the year. Please use the poll below to vote and I will do my best to work on the most voted (and not necessarily the easiest) task. So far, the poll only has tasks mentioned in recent meetings.

If you would like to suggest another task, please do so in the comments and I will add it to the poll.

**Comment by CesarCaballeroGaudes** [comment]
I would go for the second and third options as I can contribute to them

**Comment by handwerkerd** [comment]
Relating to #988, we have a bunch of open issues relating to automatically pulling and using datasets for testing. That would make a great project board that could help guide new contributors.

For the second, I think the current plan is to put the decision trees on figshare where they can get a citable DOI ( https://github.com/ME-ICA/tedana/pull/959#issuecomment-1759868141 ). I'm not sure that can be automated with GitHub Action or if we'll be putting out new decision trees so quickly that an automation would be useful. (I'm also hoping to get that done soon).

At some point I wrote up what would be necessary to  add a tsv file for regressors into the decision tree. I'll try to dig that up and turn it into an issue. I'm not sure if people who haven't already dug into the code could do this in a brainhack, but I'd support anyone who wants to try.

**Reply by tsalo** [reply]
I took a stab at making a project for the datasets: https://github.com/orgs/ME-ICA/projects/5

I also created a new repo, [`ME-ICA/open-multi-echo-data`](https://github.com/ME-ICA/open-multi-echo-data), where we can track minimally-preprocessed derivatives for open datasets. These derivatives can then be what we fetch when testing Tedana.
